packages/ncids-css/scss/settings/_settings-utilities.scss

```diff
@@ -41,8 +41,8 @@ $theme-utility-breakpoints: (
 	'mobile-lg': true,
 	// 640px:
 	'tablet': true,
-	// 800px:
-	'tablet-lg': false,
+	// 880px:
+	'tablet-lg': true,
```

Comment:
This should be reverted an moved to the index.scss for the example site. Adding tablet-lg blows up the utilities. We don't care about the CSS size for example site or storybook, so we should only turn on utilities/expand the size of utilities in those things that specifically need them.

Reply:
This has been changed.